addons/medical/functions/fnc_handleFallDamage.sqf

```diff
@@ -0,0 +1,14 @@
+// by commy2
+#include "script_component.hpp"
+
+params ["_unit", "_newDamage"];
+
+private ["_totalDamageL", "_totalDamageR"];
+
+_totalDamageL = (_unit getHitPointDamage "HitLeftLeg") + _newDamage;
+_totalDamageR = (_unit getHitPointDamage "HitRightLeg") + _newDamage;
+
+_unit setHitPointDamage ["HitLeftLeg", _totalDamageL];
+_unit setHitPointDamage ["HitRightLeg", _totalDamageR];
```

Comment:
That's true, those custom hitpoints are added to the Man class in Ace Medical's CfgVehicles.hpp, but those HitPoint serve no purpose in game. The Arma engine does not utilize them. The most they can do is serve as a fancy way of storing data in a character, similar to setVariable, which is perfectly fine for storing data to be used by the rest of ACE Medical; but causing damage to those HitPoints will not cause actual damage to a unit in game. 

You can test this by placing a player infantry unit in the editor and using the debug console to apply damage to _hitLeftLeg_ or _hitRightLeg_ and noting that the player is not inconvenienced at all and there is no outward appearance of damage. Damage _hitLegs_ and the player's movement will be restricted once damage exceeds 49% and their pants will become bloodied.